api/worker/list.go

```diff
@@ -20,6 +22,21 @@ import (
 // ---
 // produces:
 // - application/json
+// parameters:
+// - in: query
+//   name: active
+//   description: Filter workers based on active status
+//   type: string
+//   default: "true"
```

Comment:
any reason you took out the enum? it still only supports three possible values, right?

Reply:
No it supports anything really... if it doesn't parse as boolean, the filter just isn't applied

Reply:
sure, i mean "valid" values :P

Reply:
ok, what threw me off is the usage of "all" as a value in tests and the type being "string" instead of "boolean" like we do here: https://github.com/go-vela/server/blob/a7cd07fe43cd7e5e0e79f7e1ddb6ca055d518d8a/api/metrics.go#L159-L162.

Reply:
so.. `type: boolean` and take out `default: "true"` - since there is no default? if it's not supplied or a non-boolean-like string is supplied, it just doesn't apply a filter. that, or i need more coffee.

Reply:
Sounds good. Although it's hard to communicate the `all` / `both` option via the swagger docs